kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -1000,6 +1008,117 @@ def check_ceph_fencer(self):
+                clean_network_config(killed_vms.keys())
+
+    def is_fencer_private_args_change(self, cmd):
+        pass
+
+    def update_ha_fencer(self, cmd, ha_fencer):
+        pass
+
+    @bash.in_bash
+    def _fill_heartbeat_file(self, vm_uuids):
+        # type: (list[str]) -> bool
+        offset = self.host_id * self.heartbeat_required_space
```

Ruff (0.8.2)

Local variable `offset` is assigned to but never used

Remove assignment to unused variable `offset`

(F841)